src/runtime_src/core/edge/user/aie/graph.cpp

```diff
@@ -36,25 +36,16 @@ extern "C"
 }
 
 #ifdef __AIESIM__
-static zynqaie::Aie s_aie(xrt_core::get_userpf_device(0));
+static zynqaie::Aie* sp_aie = nullptr;
+zynqaie::Aie* getAieArray() {
+  if (sp_aie == nullptr)
+    sp_aie = new zynqaie::Aie(xrt_core::get_userpf_device(0));
```

Comment:
Where is this deleted?  If this like a singleton you may want to do something like this:

```
auto
getAieArray()
{
   static zynqaie::Aie s_aie(xrt_core::get_user_pf_device(0));
   return &s_aie;
}
```

Reply:
done. Thanks for the suggestion